### PR TITLE
Merge documentation

### PR DESCRIPTION
**Pull request by pepibumur**
Resolves https://github.com/tuist/tuist/issues/220

### Short description 📝

Merge the documentation from the [documentation repository](https://github.com/tuist/documentation) into this repository.

### Solution 📦

- [x] Disable documentation generation in the repository settings.
- [x] Delete the jazzy auto-generated documentation.
- [x] Delete the documentation step from the release steps.
- [x] Update the repository website to point to https://tuist.io
- [x] Remove Open Collective references.
- [x] Add a section to the README.md pointing to the documentation directory.
- [x] Copy markdown files over from the documentations repository.

### CODE REVIEW

File: docs/usage/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+To avoid that, Tuist is self-contained and comes with `tuistenv` which ensures that the right version is used. It manages different versions in your environment and runs the version your project is pinned to. Thanks to that, we ensure that anyone in your team will use the same version of Tuist.  
+In a more advanced section on the documentation, we'll see the power of `tuistenv`. For now, we'll keep things simple and just assume that we are running Tuist directly.
+
+### Creating our first project
+
+Now that we have Tuist installed, we can create our first project. Create a directory for your app:
+
+```bash
+mkdir MyAppcd MyApp
```
Comment (ollieatkinson) [inline docs/usage/getting-started.md]:
```bash
mkdir MyApp
cd MyApp
```

I think these should be on new lines

File: docs/usage/managing-versions.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+$ tuist local
+
+The following versions are available in the local environment:
+- 0.4.0
+```
+
+If you execute the same command passing a version as an argument, it pins the directory where you are running the command from to the given version:
+
+```text
+$ tuist local 0.4.0
```
Comment (ollieatkinson) [inline docs/usage/managing-versions.md]: Commands in the getting started didn't use `$` as a prefix for each bash line, do you think it's worth adding for consistency?

File: docs/usage/up.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+* **Name:** Name of the command
+* **Meet:** Command to run in the system to configure the environment.
+* **Met:** Command to run in the system to verify whether the environment is already configure. A 0 exit code means that the environment is already configured.
+
+If you have ideas of other built-in commands that Tuist could offer, don’t hesitate to [open an issue](https://github.com/tuist/tuist/issues/new) with your ideas.
+
+
+
+
+
```
Comment (ollieatkinson) [inline docs/usage/up.md]: Seems like quite a lot of extra space at the bottom of here

File: docs/usage/getting-started.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+
+```bash
+tuist init --platform ios --product application
+```
+
+The `init` command will bootstrap an iOS application, which includes the `Info.plist` files, an `AppDelegate.swift,` a tests file, and a **`Project.swift` that contains the definition of the project.**
+
+{% hint style="info" %}
+If you have used the Swift Package Manager before, the `Project.swift` file is the equivalent to the `Package.swift`.
+{% endhint %}
```
Comment (ollieatkinson) [inline docs/usage/getting-started.md]: This does not get rendered very well in markdown, maybe consider using a quote instead?

File: docs/usage/getting-started.md (hunk 1 above)
Comment (pepibumur) [reply docs/usage/getting-started.md]: Good catch 👍

File: docs/usage/managing-versions.md (hunk 2 above)
Comment (pepibumur) [reply docs/usage/managing-versions.md]: We don't really need them. I'll remove them.

File: docs/usage/getting-started.md (hunk 4 above)
Comment (pepibumur) [reply docs/usage/getting-started.md]: Agree, that's a GitBook custom syntax.

**Comment by pepibumur** [comment]
I addressed your comments @ollieatkinson

**Pull request merged**